internal/bminventory/inventory_test.go

```diff
@@ -13188,6 +13178,65 @@ var _ = Describe("TestRegisterCluster", func() {
 		}
 	})
 
+	Context("Cluster Mirror Registry", func() {
```

Comment:
Can you add a test for the infraenv functions as well as the cluster update one?

Reply:
Added